integration_tests/mockApis/manageRecallsApi.js

```diff
@@ -1,24 +1,17 @@
 export default function manageRecallsApi(wiremock) {
   return {
-    expectSearchResults: expectation => {
+    expectPrisonerResult: expectation => {
       return wiremock.stubFor({
         request: {
-          method: 'POST',
-          urlPattern: '/search',
-          bodyPatterns: [
-            {
-              equalToJson: {
-                nomsNumber: expectation.expectedSearchTerm,
-              },
-            },
-          ],
+          method: 'GET',
+          urlPattern: '/prisoner/(.*)',
```

Comment:
Outside the scope of this but probably plural is more consistent with the other endpoints